docs: refine FAQ descriptions (#3067)
beclab/Olares: docs/manual/help/olares.md

We plan to introduce new activation options that make the Olares ID optional for users who prefer to configure their own network access.

### Is internet access required to use Olares?

Olares supports local-first usage, though initial activation currently requires internet access.

For strict local control, choose these options:
- **VPN-only mode**: Restrict your Olares device so it remains accessible remotely only via VPN.
- **Local-only access**: Access Olares services via .local domains even if the router has no internet access.

For detailed local access options, refer to [Access Olares services securely](../get-started/local-access.md).

We are actively developing an option to support full device activation in completely offline environments.

### What is LarePass and why is it required?

LarePass is the official client for Olares. It acts as a secure bridge to enable seamless access, file synchronization across devices, and more. Currently, device activation requires LarePass.

### Can I use Olares without the LarePass app?

We are working on decoupling functions to support usage without the LarePass app:

> * **CLI activation**: We plan to move activation logic into the `olares-cli` to support a terminal-based setup without the app.

Reviewer: Update the wording and add a link to the doc (tutorial). But let users know that an Olares ID is still required to be created from LarePass.

> * **CLI activation**: Use the `olares-cli` tool to activate Olares directly from the terminal without using the LarePass app for the activation step. However, note that you must still use the LarePass app to create your Olares ID before running the `olares-cli` tool. For detailed instructions, see Activate an Olares device using the Olares CLI.

Reviewer: Missing the link

* **Standalone components**: We plan to provide standalone deployment options for components like the Reverse Proxy, DID service, and Market repo in future updates.

### Can I use my own domain name?

Yes. You can use your own custom domain instead of the default `olares.com` domain. Setting this up currently requires the LarePass app.

For details, refer to [Set up a custom domain for your Olares](../best-practices/set-custom-domain.md).

### Do I need to pay for Olares?

> Olares OS is free and open source for self-hosting. Purchasing Olares One requires a one-time hardware cost.

Reviewer: ```suggestion
Olares OS is free and open source for self-hosting. Olares One is a one-time hardware purchase.
```

Olares offers two optional cloud-assisted services for convenience, but free alternatives are available so you are never locked in:
* **Cloud backup**: Subscribe to Olares Space for integrated cloud backups. Alternatively, back up to your own external storage or an S3-compatible service for free.

> * **Remote access (FRP)**: Access devices remotely using the built-in Fast Reverse Proxy (FRP) service. This includes 2 GB of free monthly traffic, with paid options for higher usage. For a completely free alternative, access Olares services via LarePass VPN, or configure and use your own proxy server.

Reviewer: ```suggestion
* **Remote access**: Access devices remotely using the built-in Fast Reverse Proxy (FRP) service. This includes 2 GB of free monthly traffic, with paid options for higher usage. For a completely free alternative, access Olares services via LarePass VPN, or configure and use your own FRP server.
```

### How often does Olares update?

Olares releases a major update approximately every 2 months. View specific changes in the [Changelog](https://www.olares.com/changelog).

## License

### Is Olares open source?

Yes. The Olares OS software is open source, ensuring transparency and community collaboration. The project includes a family of repositories licensed under different models:

> * **Olares and LarePass**: Licensed under AGPL-3.0. View the [GitHub organization](https://github.com/beclab).

Reviewer: ```suggestion
* **Olares and LarePass**: Licensed under AGPL-3.0. View the [GitHub organization](https://github.com/beclab) for details.
```

## Security and privacy

### Does Olares collect my data?

No. Olares reclaims your data ownership. All storage, computation, and AI processing happen locally on your hardware. Olares does not collect or transmit your private data to any centralized service.

### Does Olares support backup?

Yes. Data safety is user-controlled and private. Olares includes a [built-in backup feature](../olares/settings/backup.md) that allows you to save specific file directories and set automatic schedules.

Every backup file is end-to-end encrypted. This allows you to store the backup file on any medium, including external drive or third-party cloud, with full confidence that the data remains inaccessible to others.

### What is app sandboxing?

Sandboxing is a security standard that prevents a single malicious app from compromising the entire system. In Olares, every app runs in its own secure, isolated environment. If an app malfunctions, it remains completely contained and fails to access or damage other applications or personal data.

### Does the system support multi-user environments?

> Yes. Olares supports sub-accounts with a built-in roles and permissions system, including Super Admin, Admin, and Member roles.

Reviewer: ```suggestion
Yes. Olares supports sub-accounts with built-in roles and permissions. The available roles are Super Admin, Admin, and Member.
```
